apps/android/app/src/main/java/chat/simplex/app/views/helpers/Util.kt

```diff
@@ -220,6 +222,11 @@ private fun spannableStringToAnnotatedString(
 // maximum image file size to be auto-accepted
 const val MAX_IMAGE_SIZE: Long = 236700
 const val MAX_IMAGE_SIZE_AUTO_RCV: Long = MAX_IMAGE_SIZE * 2
+const val MAX_VOICE_SIZE_AUTO_RCV: Long = MAX_IMAGE_SIZE_AUTO_RCV
```

Comment:
maybe it should be equal to `MAX_IMAGE_SIZE`?

Reply:
@epoberezkin decides so